dev/generate_requirements.py

```diff
@@ -18,6 +18,10 @@
 
 
 def parse_args(args):
+    """
+    Returns parsed arguments
+    :param args: Arguments to parse
+    """
```

Comment:
@KarthikKothareddy Why do we need this change?

Reply:
@harupy pylint 2.14.4 triggered warnings about methods with no docstrings in the ./dev directory.

Reply:
The category of `missing-doctsring` is C and it's disabled in this repo:
https://github.com/mlflow/mlflow/blob/f4f2d6a228f84e9bd64d65cf52af78192e90d1f2/pylintrc#L143

Reply:
@KarthikKothareddy How did you run `pylint`? Did you run it in the repository root?

Reply:
You can just fix the errors in https://github.com/mlflow/mlflow/pull/6213#issuecomment-1178550661.

Reply:
I ran them from within 'dev' directory. I just realized that the 'lint.sh' should have been run on root directory, am I correct?

Reply:
yes, it must be run on the root directory.

Reply:
Should I go back to previous commit without the 'changes' to the dev directory scripts and then take it from there?

Reply:
yes